### PR TITLE
[api] Remove yet another type alias.

### PR DESCRIPTION
**Pull request by ejgallego**

### CODE REVIEW

File: dev/doc/univpoly.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ type pinductive = inductive puniverses
 type pconstructor = constructor puniverses
 
 type constr = ...
-  | Const of puniversess
+  | Const of puniverses
```
Comment (SkySkimmer) [inline dev/doc/univpoly.txt]: Shouldn't this be pconstant?

Comment (maximedenes) [reply dev/doc/univpoly.txt]: @ejgallego ?

Comment (ejgallego) [reply dev/doc/univpoly.txt]: This is the documentation of universes, not exactly a domain I'm comfortable with.

[I regret fixing the typo now XDD]

Comment (maximedenes) [reply dev/doc/univpoly.txt]: So you mean we should merge as is, I guess?

Comment (ejgallego) [reply dev/doc/univpoly.txt]: The mistake was there before the PR, I suggest we do a pass on the file on a separate PR if needed.

**Pull request merged**